proto/chart/v1/config.proto

```diff
@@ -11,3 +12,26 @@ message UpdateChartConfigs {
   // list of config hashes missing from cloud and requested from the agent
   repeated string config_hashes = 3;
 }
+
+message ChartConfigsUpdated {
+  repeated ChartConfig configs = 1;
+}
+
+message ChartConfig {
+  string type = 1;
```

Comment:
What is the "type" here? 

The chart name in general carries (type, id and name). In the api/v1/charts you get the *id* which is *type.id* and you get the name which is **type.id** if name is not specified or **type.name** if name is valid

Reply:
Hey @stelfrag, given this sample - which was a ref point when we were talking about payloads - `type` in the proto definition is the 3rd field.

I notice that we did not include `id` and `name`. We should add them, right?

```json
{
	"id": "k8s_kubelet.kubelet_pods_log_filesystem_used_bytes",
	"name": "k8s_kubelet.kubelet_pods_log_filesystem_used_bytes",
	"type": "k8s_kubelet",
	"family": "pods",
	"context": "k8s_kubelet.kubelet_pods_log_filesystem_used_bytes",
	"title": "Bytes Used By The Pod Logs On The Filesystem (k8s_kubelet.kubelet_pods_log_filesystem_used_bytes)",
	"priority": 39907,
	"plugin": "go.d",
	"module": "k8s_kubelet",
	"enabled": true,
	"units": "B",
	"data_url": "/api/v1/data?chart=k8s_kubelet.kubelet_pods_log_filesystem_used_bytes",
	"chart_type": "stacked",
	"first_entry": 1613130647,
	"update_every": 1,
	"dimensions": {...}
}
```